documents/2-RTB/Documentazione_Interna/Norme_di_Progetto.typ

```diff
@@ -258,7 +267,320 @@ Piattaforma online che consente agli sviluppatori di creare, salvare, gestire e
 ==== #glossary("Typst")
 Linguaggio utilizzato per la creazione e la redazione della documentazione.
 
-// PARTE DI MATTEO
+== Sviluppo
+=== Introduzione
+Lo sviluppo, delineato dallo standard `ISO/IEC 12207:1995`, si occupa di organizzare ed eseguire le attività per garantire che il prodotto completo soddisfi i tutti requisiti descritti nel #glossary("contratto"). Questa sequenza include tutte le azioni intraprese dai membri del team, finalizzate alla realizzazione di un prodotto che rispecchi le esigenze e le specifiche definite dal #glossary("Proponente").
+
+Le fasi che compongono lo sviluppo sono:
+
+- *Analisi dei requisiti*
+- *Progettazione*
+- *Codifica*
+
+=== Analisi dei requisiti
+==== Descrizione
+L’Analisi dei Requisiti definisce le funzionalità che il prodotto deve offrire e i requisiti da soddisfare affinché il #glossary("software") sviluppato risulti conforme alle richieste del #glossary("proponente"). Descrive in dettaglio i casi d’uso, i #glossary("requisiti") del progetto e le funzionalità attese per il prodotto finale, basandosi sugli obiettivi definiti. Inoltre, funge da base preliminare per la progettazione del software.
+Contiene le seguenti informazioni:
+- *Descrizione del prodotto*: obiettivo finale del prodotto e le sue funzionalità principali;
+- *Lista dei casi d’uso*: identificazione di tutti gli scenari di utilizzo del sistema da parte degli utenti. Per ogni caso d’uso sono analizzati: 
+  - Scenario;
+  - Attori coinvolti;
+  - Azioni eseguibili.
+- *Lista dei requisiti*: tutte le richieste o vincoli definiti dal proponente o dedotti dal team per la realizzazione del prodotto finale. I requisiti possono essere obbligatori, desiderabili e opzionali e verranno classificati dal gruppo a seconda della loro importanza.
+
+==== Obiettivo
+L'analisi mira a ottenere una comprensione dettagliata delle necessità degli utenti, delle caratteristiche del prodotto e delle condizioni operative previste.
+
+Questa fase include tre aspetti principali:
+
+- *Identificazione* degli *obiettivi* e delle *finalità* del prodotto: Definizione chiara e condivisa degli scopi e dei risultati attesi, tenendo conto delle esigenze degli utenti e delle proposte dell’azienda.
```

Comment:
296: **identificazione** degli **obiettivi** e delle **finalità** ->  metterei tutto in grassetto